### ISSUE TITLE
xll got auto-opened in Excel UI

### ISSUE THREAD

**Issue by 4R3B3LatH34R7** [Excel-DNA/IntelliSense#108]
Office365 Excel 64bit with Excel-DNA Intellisense 1.4.2.xll, Win10 64bit.
Unblocked already in C: users addins directory.
When .xlsm was opened, xll was auto-opened as readonly in UI as well, with every xlsx and xlsm.
Cannot uncheck the addin from Developer - Addins menu.
Had to manually delete the .xll.
Error message as shown in attached photo.
![excel-dna intellisense error message](https://user-images.githubusercontent.com/53028471/147851826-51c56cc6-4b0e-45db-b837-211491f63ca7.png)

**Comment by govert** [comment]
That error message normally means that you are opening the 64-bit version of the add-in in a 32-bit installation of Excel (or vice versa).

**Comment by govert** [comment]
Maybe you can show a screenshot of the Excel version (File -> Account -> About Excel). I have this:
![image](https://user-images.githubusercontent.com/414659/147853034-e78b58fa-371b-40f0-87ad-b4c3542facff.png)

**Comment by 4R3B3LatH34R7** [comment]
Yes, you are right, my user was quite adamant that her Office365 was 64bit.
Problem was solved by using 32bit xll.

The only issue left was: if .xlsm(containing _Intellisense_ worksheet) was double clicked to open, the Intellisense doesn't work for UDF and needs to close the file from File menu then open .xlsm again to get Intellisense working.
However, if Excel were run first, then open .xlsm from recent list, Intellisense kicked in immediately.
Excel-DNA Intellisense host was checked in Developer-Addins in both scenarios.
Thank you.
